Libraries/Components/ScrollView/ScrollView.windows.js

```diff
@@ -1,11 +1,14 @@
 /**
  * Copyright (c) 2015-present, Facebook, Inc.
- * All rights reserved.
  *
```

Comment:
Detailed review needed.

Reply:
RNTester still working properly.

---
In reply to: [169646847](https://github.com/Microsoft/react-native-windows/pull/1659#discussion_r169646847) [](ancestors = 169646847)